### PR TITLE
Added text to Initiatives board about traceability of sub tasks

### PR DESCRIPTION
**Pull request by Gman0909**

### CODE REVIEW

File: cross-functional/workflow.md
Comment (aprilmj) [inline cross-functional/workflow.md:1]: Suggest a slight addition - "be traceable from the initiative card on the board _and the goal the initiative serves_.

**Review by aprilmj — APPROVED** [review]

File: cross-functional/workflow.md [hunk 1]
```diff
@@ -36,6 +36,20 @@ There are three types of items on the initiatives board:
   - Investment initiatives are an investment in our team, technology, etc that does not immediately deliver value to communities, but helps us build a better foundation to do so.
 - **Lists of Tasks and Epics**: Are the sequence of steps or deliverables that are taken to complete an initiative. These are defined as `[tasklist]` objects in each initiative, and are the main units of work that drive our actions.
 
+:::{admonition} What about community projects?
```
Comment (choldgraf) [inline cross-functional/workflow.md:39]: Just moved this upward since I'm creating a new section

File: cross-functional/workflow.md [hunk 2]
```diff
@@ -36,6 +36,20 @@ There are three types of items on the initiatives board:
...
+
+We also track community projects as initiatives.
+When 2i2c is actively engaged in work for upstream projects or communities we serve, we add those to our initiatives list, and follow the planning process outlined here. 
+
+Note this is a work in progress, and we’re continuing to evolve our process for collaborating within and outside 2i2c.
+:::
+
+### Structure of an initiative
```
Comment (choldgraf) [inline cross-functional/workflow.md:47]: Created a new section so that this information is easier to spot

File: cross-functional/workflow.md [hunk 3]
```diff
@@ -65,14 +79,16 @@ what kind of work it will entail, and why it is important to 2i2c's overall goal
...
-We also track community projects as initiatives.
-When 2i2c is actively engaged in work for upstream projects or communities we serve, we add those to our initiatives list, and follow the planning process outlined here. 
+`In Progress` initiatives should have a **task list of steps to complete the initiative**.
+These become the issues that we plan in [Delivery Boards](#board:delivery) and iteration planning.
 
-Note this is a work in progress, and we’re continuing to evolve our process for collaborating within and outside 2i2c.
-:::
+Ensure that every initiative has the right `[tasklist]` items and linked issues so that:
+
+1. Every task can be traced back to its parent initiative (by clicking upward through parent issues)
```
Comment (choldgraf) [inline cross-functional/workflow.md:88]: Split this into a two item list to make it explicit

Comment (choldgraf) [inline cross-functional/workflow.md:88]:
```suggestion
1. Every task can be traced back to its parent initiative by clicking upward through parent issues.
```

**Review by choldgraf — APPROVED** [review]
I pushed a few changes to make the information easier to find, it looks good to me. Thanks for following up with this improvement @Gman0909 !

[Link to preview](https://2i2c-team-compass--852.org.readthedocs.build/cross-functional/workflow/#structure-of-an-initiative)

I'll hold off merging in case @yuvipanda wants to share thoughts, but suggest we merge tomorrow if there's no objection by then

**Comment by Gman0909** [comment]
Good changes, thanks Chris.

**Comment by choldgraf** [comment]
let's get this one in, thanks @Gman0909 !

**Pull request merged**